src/main/java/edu/bu/metcs673/trackr/security/WebSecurityConfig.java

```diff
@@ -56,7 +56,15 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
 	 */
 	@Override
 	protected void configure(HttpSecurity http) throws Exception {
-		http.csrf(csrf -> csrf.csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse()))
+		http
+			.csrf(csrf -> {
+					// enable CSRF protection, ignore the following paths which are entry/exit
+					// points, or API paths
+					csrf.csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse())
+					.ignoringAntMatchers("/api/**")
```

Comment:
We don't CSRF because we are using the JWT, is that correct?

Reply:
yes